server/oauth-webapp/src/main/java/org/eclipse/lyo/server/oauth/webapp/services/CSRFPrevent.java

```diff
@@ -19,18 +19,23 @@
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.Response.Status;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  * Checks requests to see if they have the right X-CSRF-Prevent header values. 
  * 
  * @author Samuel Padgett
  */
 public class CSRFPrevent {
 	private static final String CSRF_PREVENT_HEADER = "X-CSRF-Prevent";
-	
-	public static void check(HttpServletRequest httpRequest) {
+    private static final Logger log = LoggerFactory.getLogger(CSRFPrevent.class);
+
+    public static void check(HttpServletRequest httpRequest) {
 		String csrfPrevent = httpRequest.getHeader(CSRF_PREVENT_HEADER);
 		String sessionId = httpRequest.getSession().getId();
 		if (!sessionId.equals(csrfPrevent)) {
+		    log.error("Request denied due to possible CSRF attack. Expected X-CSRF-Prevent header: {}. Received: {}", sessionId, csrfPrevent);
```

Comment:
Same reasoning as above: during a DDoS attack or simple overload, you may want to set the log level to ERROR. Plus this does not seem an application error.

Reply:
Not sure what you are suggesting here. It is already "error".

Reply:
I meant that we may want to consider setting the level of this to "WARN" so that if under attack, we can set the log level to ERROR only and avoid the logging system from being overwhelmed.

Reply:
fixed